pkg/metrics/bindinfo.go

```diff
@@ -18,16 +18,48 @@ import "github.com/prometheus/client_golang/prometheus"
 
 // bindinfo metrics.
 var (
-	BindUsageCounter *prometheus.CounterVec
+	BindingCacheHitCounter  prometheus.Counter
+	BindingCacheMissCounter prometheus.Counter
+	BindingCacheMemUsage    prometheus.Gauge
+	BindingCacheMemLimit    prometheus.Gauge
+	BindingCacheNumBindings prometheus.Gauge
 )
 
 // InitBindInfoMetrics initializes bindinfo metrics.
 func InitBindInfoMetrics() {
-	BindUsageCounter = NewCounterVec(
+	BindingCacheHitCounter = NewCounter(
 		prometheus.CounterOpts{
 			Namespace: "tidb",
-			Subsystem: "bindinfo",
-			Name:      "bind_usage_counter",
-			Help:      "Counter of query using sql bind",
-		}, []string{LabelScope})
+			Subsystem: "server",
+			Name:      "binding_cache_hit_total",
+			Help:      "Counter of binding cache hit.",
+		})
+	BindingCacheMissCounter = NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "server",
+			Name:      "binding_cache_miss_total",
```

Comment:
```suggestion
            Subsystem: "binding",
			Name:      "cache_miss_total",
```

Reply:
subsystem should be ```binding```. name can remove the ```binding``` string.

Reply:
Want to keep it in the `server` subsystem, same as Plan Cache.

Reply:
OK